src/main/java/org/torusresearch/torusutils/helpers/KeyUtils.java

```diff
@@ -122,7 +125,7 @@ public static String[] getPublicKeyCoords(@NotNull String pubKey) throws TorusUt
         if (publicKeyUnprefixed.length() <= 128) {
             Common.padLeft(publicKeyUnprefixed, '0', 128);
         } else {
-            throw new TorusUtilError("Invalid public key size");
+            throw new TorusUtilError("Invalid public key size", TorusUtils.getClientId());
```

Comment:
Perhaps you can promote this to a full error, then it can sit will all the other errors. Will be easy to keep track of then, simillary for the others like this.